core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -48,7 +52,17 @@ object KafkaConsumerActor {
 
     private[KafkaConsumerActor] class NoPollResult extends RuntimeException with NoStackTrace
   }
-
+  
+  /** Public protocol which can be used to interact with the Actor */
+  object Protocol {
+    // requests
+    final case class RequestMetrics()
```

Comment:
strange with case class without params, case object?

then it need accessor for Java API

Reply:
ok